_episodes/04-changes.md

```diff
@@ -19,62 +19,39 @@ keypoints:
 ---
 
 First let's make sure we're still in the right directory.
-You should be in the `planets` directory.
+You should be in the `metadata` directory.
 
 ~~~
 $ pwd
 ~~~
 {: .bash}
 
-If you are still in `moons` navigate back up to `planets`
+If you are still in `csv` navigate back up to `metadata`
 
 ~~~
 $ cd ..
 ~~~
 {: .bash}
 
-Let's create a file called `mars.txt` that contains some notes
-about the Red Planet's suitability as a base.
-We'll use `nano` to edit the file;
-you can use whatever editor you like.
-In particular, this does not have to be the `core.editor` you set globally earlier. But remember, the bash command to create or edit a new file will depend on the editor you choose (it might not be `nano`). For a refresher on text editors, check out ["Which Editor?"](https://swcarpentry.github.io/shell-novice/03-create/) in [The Unix Shell](https://swcarpentry.github.io/shell-novice/) lesson.
+Let's add a file called `readme.md` that contains some notes about our
+metadata.
 
-~~~
-$ nano mars.txt
-~~~
-{: .bash}
-
-Type the text below into the `mars.txt` file:
-
-~~~
-Cold and dry, but everything is my favorite color
-~~~
-{: .output}
-
-`mars.txt` now contains a single line, which we can see by running:
-
-~~~
-$ ls
-~~~
-{: .bash}
-
-~~~
-mars.txt
-~~~
-{: .output}
+We'll use `nano` to edit the file for these examples, but you can use
+whatever editor you like.  In particular, this does not have to be the
+`core.editor` you set globally earlier. But remember, the bash command
+to create or edit a new file will depend on the editor you choose (it
+might not be `nano`). For a refresher on text editors, check out
+["Which
+Editor?"](https://swcarpentry.github.io/shell-novice/03-create/) in
+[The Unix Shell](https://swcarpentry.github.io/shell-novice/) lesson.
 
 ~~~
-$ cat mars.txt
+$ cp /path/to/this/repo/git-novice/files/readme.md ../metadata/
```

Comment:
So what's the idea w/ `cp` here? I'm not following.

Reply:
Since we want them to create their own git repo instead of just modifying this one, the idea was that they'd copy the starter files we added from here to wherever their new repo is.

Reply:
Ah, ok. Yeah I see what you're doing. I guess an alternative is we make a separate starter repo they can clone and then add commits to. No strong opinions though. This totally works well.

Reply:
If we stick with this we'll have to make it clearer.